### DISCUSSION TITLE
Delayed start?

### DISCUSSION THREAD

**Discussion by TfTHacker** [st3v3nmw/obsidian-spaced-repetition#146] [Q&A]
Is it possible  to delay the start of the spaced repetitions? I ask because I like  to practice something I am learning for a few days, then want to stick it into the SR system, but have it come up not  for at least a few days, to allow my to test the initial strength of the info?

I see I could do a test, then modify the date, but was curious if there was another way.

**Comment by st3v3nmw** [comment]
Hello, sorry for the delayed response,
I've thought about this & it fits into [my vision](https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/154) of what should be the next version of this plugin i.e. moving scheduling information away from the notes themselves and into a dedicated JSON file. Which will give us the flexibility that could enable us to have this delayed start functionality. If I understand you correctly, I think it could be done using something like [Anki's learning steps](https://eshapard.github.io/anki/what-anki-learning-steps-to-use.html) (longer learning steps in this case).

Edit:
On the "vision" => The current implementation will remain as is. Cause conflicts & portability!

**Comment by TfTHacker** [comment]
Thank you for this article. Basically “Learning Steps” is what I want to do — but didn’t know this as a part of Anki. Some automated process of helping that initial learning. I have found for myself I have to review something for a few days, but its hard to manage and get it into SR. Currently I am exploring some tagging and a dataview / embedded search to handle this.

Learning steps would be a welcome addition to your tool, thought I can see that is a lot for work.

Regarding saving the testing data to a central JSON file, I would like to give my opinion. Not saying its right, but perhaps of value. I actually like the current method that the testing data is embedded in the file. First, with file syncing, lot less worry for potential conflict than having one large file. Second, I chose your plugin just for the approach you are using. To me it makes the reviews portable, which is one of the goals of markdown. So if I import this file later into something like Roam, then I have the current state of the testing as part of the note file, and can adapt another SR system in the new environment.

The one thing I do hope for is that in time there is a standard way for inline metadata. Your approach with HTML comments is brilliant, as the testing data is not visible in HTML view.  But for future parsing, with tools like dataview, if inline meta data could be used, it might make it more usable between plugins.

P.S. Few days back sent you some coffee money. Very appreciative of your work. I have been reviewing your code and would like to contribute, but still trying to get the feel for the system and also your code.

**Reply by st3v3nmw** [reply]
Thanks for this. I'll list the portability and conflicts disadvantages in the article.
The conflicts bit hit home this morning after I made some changes to some cards on desktop, forgot to sync, reviewed on mobile, then had to resolve a multitude of merge conflicts. That would be hella nasty in a central json file full of UUIDs.
I guess we'll stick with the current implementation and find workarounds to its limitations.

**Comment by christopherAI2022** [comment]
I too am definitely interested in a learning steps kind of solution!  There has to be some kind of workaround.

Perhaps a third category?  Learning.

#flashcards #notes.  With an automatic adding of a #review tag.

Things tagged appropriately can have their own field in the settings that describes the steps in minutes (or seconds).  

Then they automatically get tagged with #review and any cards that have #review skip the learning steps.
